genomeGraph/toGiraf_test.go

```diff
@@ -1,26 +0,0 @@
-package genomeGraph
-
-/*
-import (
```

Comment:
Commented out tests, lets just delete this file and we can always revert or roll back